kolibri/core/assets/src/core-actions.js

```diff
@@ -0,0 +1,62 @@
+function kolibriLogin(store, Kolibri, sessionPayload) {
+  const SessionResource = Kolibri.resources.SessionResource;
+  const sessionModel = SessionResource.createModel(sessionPayload);
+  const sessionPromise = sessionModel.save(sessionPayload);
+  const UserKinds = require('./constants').UserKinds;
+  sessionPromise.then((session) => {
+    store.dispatch('CORE_SET_SESSION', session);
+    if (session.kind.includes(UserKinds.ADMIN) || session.kind === UserKinds.SUPERUSER) {
+      store.dispatch('SET_ADMIN_STATUS', true);
+    }
+  }).catch((error) => {
+    // hack to handle invalid credentials
+    if (error.status.code === 401) {
+      store.dispatch('HANDLE_WRONG_CREDS', { kind: 'ANONYMOUS', error: '401' });
+    } else {
+      store.dispatch('CORE_SET_ERROR', JSON.stringify(error, null, '\t'));
+    }
+  });
+}
+
+function kolibriLogout(store, Kolibri) {
+  const SessionResource = Kolibri.resources.SessionResource;
+  const id = 'current';
+  const sessionModel = SessionResource.getModel(id);
+  const logoutPromise = sessionModel.delete(id);
```

Comment:
good point, we don't actually need to pass the id here, changed here https://github.com/learningequality/kolibri/pull/359/files